### ISSUE TITLE
Medusa can not search/process shows starting with "True"

### ISSUE THREAD

**Issue by hagabla** [pymedusa/Medusa#2828]
I am having trouble with the show "True Detective", medusa is unable to find any episodes or manual postprocessing it when downloaded manually. It seems medusa skips the "True" in "True Detective"
Here is part of a log, notice the "title: Detective" part
```
Unable to match True.Detective.S02E05.1080p.HDTV.x264-BATV to a
series in your database. Parser result: language: und, title:
Detective, season: 2, episode: [5], screen_size: 1080p, format:
HDTV, video_codec: h264, video_encoder: x264, release_group:
BATV, type: episode, parsing_time: 0.0194780826569, total_time:
0.0536179542542, absolute_episode: [], quality: 1080p HDTV
```

I tested it and added the show "True Blood" with the same result.

**Comment by ratoaq2** [comment]
This issue was a guessit issue : https://github.com/guessit-io/guessit/issues/434

It should be already fixed in our development branch. 
In which version are you?

**Comment by hagabla** [comment]
Latest master

**Comment by hagabla** [comment]
Could this be merged to master?

**Comment by fernandog** [comment]
Maybe this weekend

**Comment by hagabla** [comment]
Appreciated.

**Comment by medariox** [comment]
Fixed with release 0.1.13
